calcom/cal.com
# refactor: add env creation on yarn dx

- build: add script to create new .env files
- build: add env script creation on turbo dx

## What does this PR do?



Fixes #8591

/claim #8591



## Type of change



- Chore (refactoring code, technical debt, workflow improvements)
- This change requires a documentation update

## How should this be tested?



- [ ] **[CASE-0]** Testing that it'll not affect what is already running
  - Get your repo that you already spin up (you already have the configs and don't need to create any envs/seed anything)
  - Run the app as you would do normally
  - These changes should not affect what already works.
- [ ] **[CASE-1]** Testing in a brand-new repo
  - Clone the repo
  - Install the dependencies using `yarn`
  - Run the app using `yarn dx`
- [ ] **[CASE-2]** Testing in a repo that already exists
  - Run `git clean -dX -f` (this will delete everything on your repo that git does not track)
  - Install the dependencies using `yarn`
  - Run the app using `yarn dx`

## Checklist



- I haven't checked if my PR needs changes to the documentation
